### ISSUE TITLE
Inconsistencies in token handling, for code samples.

### ISSUE THREAD

**Issue by wiedld** [influxdata/ui#3772]
## Summary

We have a deviation in the token handling for the executeFull template, versus the initialize template. This inconsistency exists in the python templates, but should also be checked for other client libraries.

### Error in files:
Per client lib defined in `src/writeData/<client>`:
- each has an initialize template, and an executeFull template.
- in some cases, these templates use a token held in the app state. (With a default value set in the app state.)

### Specific example, in one language client:
In python:
- the initialize template uses the env variable `token = os.getenv("INFLUX_TOKEN")`.
- the executeFull template uses the token in the running app state, and the default is `<INFLUX_TOKEN>` (see `src/writeData/components/WriteDataDetailsContext.tsx`).

I would suggest making the default be the same (per syntax/client). For python this would be:
`os.getenv("INFLUX_TOKEN")`

Also, make sure to update handling of the string quotes a bit differently. Such that for each syntax/client, and per if using the app's state token or the the default, it returns the correct quotes.

### Expected result:
In summary, the resultant output code (per each language) should be consist in all cases of:
- executeFull template, versus initialize template
- when the token is provided in the app state, versus the default set in the app state. (See `WriteDataDetailsContext.tsx`.)

### To reproduce the bug
1. go to notebooks. create a flux query, and export to client Python. will see the token handled as `token = "<INFLUX_TOKEN>"` 
2. go to the client libraries page. select the python option. see the example initialization query, which uses `token = os.getenv("INFLUX_TOKEN")`
3. Repeat this comparison: for notebooks export to client code, versus the client libraries page. For all languages and syntax.

**Comment by barbaranelson** [comment]
Good catch. We just changed the client library pages to use the environment variable. (They used to let you pick a token and then they would populate the actual token string into the code snippet which was a security problem.) We should make the same changes in the notebooks page, to reference the environment variable.

**Comment by drdelambre** [comment]
https://analytics.amplitude.com/influxdb/chart/9gndzkq
